src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -247,7 +254,7 @@ private sealed class Mapper<TCalibrator> : MapperBase, ISaveAsOnnx
                 _scoreColumnName = scoreColumnName;
                 _scoreColIndex = inputSchema.GetColumnOrNull(_scoreColumnName)?.Index ?? -1;
 
-                parent.Host.Check(_scoreColIndex >= 0, "The data to calibrate contains no 'Score' column");
+                parent.Host.Check(_scoreColIndex >= 0, @"The data to calibrate contains no \'{scoreColumnName}\' column");
```

Comment:
nit: maybe you meant "$" instead of "@"?

![image](https://user-images.githubusercontent.com/38739674/86285565-af14fc80-bb99-11ea-93fb-7e0638691117.png)